Submission of Lab 02 by BT23F05F067

gangaprasad29 opened the pull request:
I've Completed my  lab,
by solving  merge_sort_basic.py on GitHub codespace.

gangaprasad29 committed:
Complete merge_sort_basic.py

I've Completed my  lab,
by solving  merge_sort_basic.py on GitHub codespace.

s-m-quadri commented:
Hi @bot-s-m-quadri, give us the status.

bot-s-m-quadri commented:
### PRN: <a href="https://github.com/s-m-quadri/geca-labs/pulls?q=BT23F05F067" target="_blank"><code>BT23F05F067</code></a> Status

Please complete the current lab before the review date.

> [!NOTE]
> - Submission Date: **31 August 2025, 11:21 AM IST**  
> - Next Review: **07 September 2025**  
>
> | Info. | Value/Link |
> |-----------------:|:------|
> | PRN              | <a href="https://github.com/s-m-quadri/geca-labs/pulls?q=BT23F05F067" target="_blank"><b>BT23F05F067</b></a> (Click to search all PR) |
> | Lab Number       | <a href="https://www.s-m-quadri.me/geca/daa/02" target="_blank"><b>02</b></a> (Click to open manual pages) |
> | Base Repository  | <a href="https://github.com/s-m-quadri/geca-labs" target="_blank"><b>geca-labs</b></a> (Click to open repository) |
> | Base Branch      | <a href="https://github.com/s-m-quadri/geca-labs/tree/stable" target="_blank"><b>stable</b></a> (Click to view problem set) |
> | Student's (Merge) Repository | <a href="https://github.com/gangaprasad29/geca-labs" target="_blank"><b>gangaprasad29/geca-labs</b></a> (Click to view student fork) |
> | Student's (Merge) Branch     | <a href="https://github.com/gangaprasad29/geca-labs/tree/lab-daa-02" target="_blank"><b>lab-daa-02</b></a> (Click to <b>resume</b> your work) |

### All Submissions
- #193
- #192

In case of doubts, contact 📧 **dev.smq@gmail.com**, or <a href="https://github.com/s-m-quadri/geca-labs/pull/192" target="_blank"><b>comment here</b></a>.

Thanks for your initiation of PR! 🙂 Keep it up!

gangaprasad29 committed:
Complete merge_sort_descending.py

I've Completed my  lab,
by solving merge_sort_descending.py on GitHub codespace.

gangaprasad29 committed:
Complete merge_sort_strings.py

I've Completed my  lab,
by solving merge_sort_strings.py on GitHub codespace.

gangaprasad29 committed:
Complete task_count_inversions.py

I've Completed my  lab,
by solving task_count_inversions.py on GitHub codespace.

gangaprasad29 committed:
Complete python task_merge_sorted_lists.py

I've Completed my  lab,
by solving python task_merge_sorted_lists.py on GitHub codespace.

gangaprasad29 committed:
Complete task_sort_student_scores.py

I've Completed my  lab,
by solving python task_sort_student_scores.py on GitHub codespace.

s-m-quadri commented:
:+1: Good. Seems fine to me.

@bot-s-m-quadri merge it now.

bot-s-m-quadri commented:
👍 Got it, I’ll start merging and organizing your submission.
